views/fxa_authorize.js

```diff
@@ -25,9 +25,10 @@ define('views/fxa_authorize', ['capabilities', 'log', 'login', 'utils', 'z'],
             var packaged_origin = 'app://packaged.' + window.location.host;
             console.log('Sending OAuth code to parent window ' +
                         packaged_origin);
+            // Don't remove the splash screen for this view;
+            z.context.dont_hide_splash = true;
```

Comment:
nit: I find double negative variables hard to read. This would read easier to me as: ``context.hide_splash = false``